person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
Great job @Matthieu96Code 🤝

- [ ] Kindly ensure Getters and Setters for @rentals should be present.